src/redist/project.json

```diff
@@ -19,12 +19,11 @@
     "tool_nuget": "1.0.0-preview3-*",
     "tool_msbuild": "1.0.0-preview3-*",
 
-    "MSBuild": "0.1.0-preview-00043-160929",
-    "Microsoft.Build.Framework": "0.1.0-preview-00043-160929",
-    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00043-160929",
-    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00043-160929",
-    "Microsoft.Build.Targets": "0.1.0-preview-00043-160929",
-    "Microsoft.Build": "0.1.0-preview-00043-160929",
+    "Microsoft.Build.Runtime": "15.1.298-preview5",
```

Comment:
Yes I would recommend if you don't have any compile-time references that you only add `Microsoft.Build.Runtime`.  It's dependencies include our other binaries.